model/histogram/float_histogram.go

```diff
@@ -307,13 +307,17 @@ func (h *FloatHistogram) Sub(other *FloatHistogram) *FloatHistogram {
 // Exact match is when there are no new buckets (even empty) and no missing buckets,
 // and all the bucket values match. Spans can have different empty length spans in between,
 // but they must represent the same bucket layout to match.
+// Sum, Count, and ZeroCount are compared based on their bit patterns because this method
```

Comment:
The buckets need to be mentioned here, too.

Reply:
My bad, I always forget those comments. Updated.